src/app/treelist/treelist.component.scss

```diff
@@ -0,0 +1,79 @@
+@import "../../assets/stylesheets/_custom";
+
+.tree-list {
+  .node-content-wrapper {
+    display: inherit;
+    padding: 0;
+    border-radius: inherit;
+    transition: none;
+  }
+
+  /*
+  // Todo: drop area
+  .node-drop-slot {
+    height: 2px;
+  }
+  .node-drop-slot.is-dragging-over {
+    height: 55px;
+  }
+  */
+
+  .toggle-children-wrapper {
+    padding: 0px;
+  }
+
+  // Toggle up
+  .toggle-children {
+    background-image: none;
+    background-position: inherit;
+    background-repeat: inherit;
+    display: block;
+    font-family: FontAwesome;
```

Comment:
The problem is that "\f105" won't match anything else. I checked Patternfly and they don't have a default either. 

.secondary-nav-item-pf {
  > a {
    cursor: default;
    &:after {
      content: @fa-var-angle-right;
      font-family: "FontAwesome";